docs/grid.html

```diff
@@ -0,0 +1,270 @@
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='grid-alt' type='solid' color='#eeeded' ></box-icon>
+                            <span>Grids</span>
+                        </li>
+                       </a>
+                       <a href="../docs/image.html" class="text-deocration-none selected-text ">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='images' color='#eeeded' ></box-icon>
+                            <span>Images</span>
+                        </li>
+                       </a>
+                       <a href="../docs/list.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='list-ul' color='#eeeded' ></box-icon>
+                            <span>Lists</span>
+                        </li>
+                       </a>
+                       <a href="../docs/modal.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='card' color='#eeeded' ></box-icon>
+                            <span>Modal</span>
+                        </li>
+                       </a>
+                       <a href="../docs/navigation.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='navigation' color='#eeeded' ></box-icon>
+                            <span>Navbars</span>
+                        </li>
+                       </a>
+                       <a href="../docs/rating.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='star-half' type='solid' color='#eeeded' ></box-icon>
+                            <span>Ratings</span>
+                        </li>
+                       </a>
+                       <a href="../docs/toast.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='copy' color='#eeeded' ></box-icon>
+                            <span>Toasts</span>
+                        </li>
+                       </a>
+                       <a href="../docs/typography.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='font-family' color='#eeeded' ></box-icon>
+                            <span>Typography</span>
+                        </li>
```

Comment:
fix: span tags are non semantic, you should use p here and make them display:inline block;
Why: It is because span tags are non semantic so they would be omiited by screen readers, and you have important information in span tag. Each span tag is containing the name of the component which is important for users

Reply:
> fix: span tags are non semantic, you should use p here and make them display:inline block; Why: It is because span tags are non semantic so they would be omiited by screen readers, and you have important information in span tag. Each span tag is containing the name of the component which is important for users

@Ankur9669 totally agree with accessibility part but p tag has by default 1em margin-top and margin-bottom which i dont wanted margin by default. yes i can change the margin by making specific css class that would also work. sure will do the changes in next iteration of CL.